### PR TITLE
fix: styling of dropdown component

### LINKED ISSUE #6417: [CAL-757] UI Dropdown menu's are inconsistent all over the app
It's all detailed in Figma: 

[   ↳ Dropdown menu - Cal - Live (Figma)](https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=1359%3A151225&t=3hNynQ6hBKZ73ols-1)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-757](https://linear.app/calcom/issue/CAL-757/ui-dropdown-menus-are-inconsistent-all-over-the-app)</sub>

### PR DESCRIPTION
**Pull request by Udit-takkar**
## What does this PR do?

Fixes https://github.com/calcom/cal.com/issues/6417

Screenshot 2023-01-13 at 1 28 32 AM

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Chore (refactoring code, technical debt, workflow improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Review by sean-brydon — APPROVED** [review]
These changes LGTM -

Can we ensure all dropdowns match the spec in figma. When we were looking at the app today a few of them look vastly different from each other. This can happily be picked up in another PR if required

**Review by PeerRich — COMMENTED** [review]
CleanShot 2023-01-12 at 21 11 15@2x

does this supposed to have double borders?


also should the delete button not hover in read? @Jaibles 
CleanShot 2023-01-12 at 21 11 53@2x

**Comment by sean-brydon** [comment]
> CleanShot 2023-01-12 at 21 11 15@2x
> 
> does this supposed to have double borders?
> 
> also should the delete button not hover in read? @Jaibles CleanShot 2023-01-12 at 21 11 53@2x

Yes this should be using destructive button

File: packages/ui/components/form/dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const DropdownMenuContent = forwardRef<HTMLDivElement, DropdownMenuConten
         align={align}
         {...props}
         sideOffset={sideOffset}
-        className="shadow-dropdown w-50 relative z-10 origin-top-right rounded-md border border-gray-200 bg-white text-sm"
+        className="shadow-dropdown w-50 relative z-10 origin-top-right rounded-md border-2 border-gray-200 bg-white text-sm"
```
Comment (PeerRich) [inline packages/ui/components/form/dropdown/Dropdown.tsx:39]:
```suggestion
        className="shadow-dropdown w-50 relative z-10 origin-top-right rounded-md border border-gray-200 bg-white text-sm"
```

**Comment by PeerRich** [comment]
CleanShot 2023-01-12 at 21 16 59@2x
 
i think we also wanna get rid of the border on hover right?

**Comment by Udit-takkar** [comment]
@sean-brydon this should fix all dropdown as we are making changes in the shared component. is there any dropdown you found not consistent ?

**Comment by Udit-takkar** [comment]
yes @PeerRich

**Comment by sean-brydon** [comment]
> @sean-brydon this should fix all dropdown as we are making changes in the shared component. is there any dropdown you found not consistent ?

image
https://app.cal.com/event-types/[type]

image
https://app.cal.com/teams 

both have weird hover states applied to them. I know there was a few more  @Jaibles mentioned in the figma file

**Comment by Udit-takkar** [comment]
@sean-brydon  okay this file right https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=1359%3A151229&t=f5YPf9kBBKMytWmP-0 ?

**Comment by sean-brydon** [comment]
> @sean-brydon okay this file right https://www.figma.com/file/xk4HOxtSI82J0F7enMxeak/Cal---Live?node-id=1359%3A151229&t=f5YPf9kBBKMytWmP-0 ?

Thats the one! @Udit-takkar Thanks a lot great work so far - feel like this is huge part of the app that looks inconstant

**Comment by Jaibles** [comment]
> does this supposed to have double borders?
No double borders in our app, anywhere, ever :)

There's an overlap issue here where the menu item is not being clipped
CleanShot 2023-01-12 at 21 00 33@2x

**Comment by Udit-takkar** [comment]
Screenshot 2023-01-13 at 11 03 40 PM
Screenshot 2023-01-13 at 11 04 08 PM

+ more

cc: @sean-brydon

**Comment by PeerRich** [comment]
NIT:  delete in availabilty is missing the delete icon

CleanShot 2023-01-14 at 01 07 13@2x

**Comment by PeerRich** [comment]
styling of dropdown in routing forms still broken 

https://user-images.githubusercontent.com/8019099/212439853-258ebbf7-7591-43db-aaa1-d660dbd7c5a3.mp4

**Comment by Udit-takkar** [comment]
1. @PeerRich in the availability design there was no trash icon for delete option. Should I add it?

2. Will fix this in routing forms

**Pull request merged**